TestE2E/PAR/CONFIG/config_file/MaxLoopDevicesOK  failure on some systems

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.6.0-rc.4+13-g31aac78a8
```

### Actual behavior

On Ubuntu 20.04 and Fedora 32

```
--- FAIL: TestE2E (368.22s)
    --- FAIL: TestE2E/PAR (0.00s)
        --- FAIL: TestE2E/PAR/CONFIG (0.00s)
            --- FAIL: TestE2E/PAR/CONFIG/config_file (9.19s)
                --- FAIL: TestE2E/PAR/CONFIG/config_file/MaxLoopDevicesOK (0.87s)
```

Does not fail on RHEL7. Looks like a new kernel issue.

--- a/e2e/config/config.go
+++ b/e2e/config/config.go
@@ -421,7 +421,7 @@ func (c configTests) configFile(t *testing.T) {
 			name:    "MaxLoopDevicesOK",
 			argv:    []string{c.env.ImagePath, "true"},
 			profile: e2e.RootProfile,
-			conf:    "max loop devices = 10\n",
+			conf:    "max loop devices = 128\n",
 			exit:    0,
 		},
 		{

